internal/install/install.go

```diff
@@ -17,30 +17,34 @@ import (
 	"gopkg.in/yaml.v2"
 
 	"github.com/newrelic/newrelic-cli/internal/credentials"
+	"github.com/newrelic/newrelic-client-go/newrelic"
 )
 
-func install(configFiles []string) error {
+func install(client *newrelic.NewRelic) error {
+	ctx := getSignalContext()
+	rf := newServiceRecipeFetcher(&client.NerdGraph)
+	pf := newRegexProcessFilterer(rf)
+
 	// Execute the discovery process.
 	log.Debug("Running discovery...")
-	var d discoverer = new(psUtilDiscoverer)
-	manifest, err := d.discover()
+	var d discoverer = newPSUtilDiscoverer(pf)
```

Comment:
Using DI to inject a `regexProcessFilterer` instance into a `psUtilDiscoverer` instance.